packages/synapse-react-client/src/components/MoreResources/MoreResources.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, Stack, Typography, useTheme, Link } from '@mui/material'
+import { Link as RouterLink } from 'react-router'
+
+type ResourceConfig = {
+  title: string
+  description: string
+  link: string
+}
+
+export type MoreResourcesProps = {
+  resources: ResourceConfig[]
+}
+
+const MoreResources = ({ resources }: MoreResourcesProps) => {
+  const theme = useTheme()
+  return (
+    <Box
+      sx={{
+        padding: { xs: '40px', lg: '60px 80px 80px 80px' },
+        background: `linear-gradient(0deg, rgba(255, 255, 255, 0.90) 0%, rgba(255, 255, 255, 0.90) 100%), ${theme.palette.primary.main}`,
+      }}
+    >
+      <Typography
+        sx={{
+          fontSize: '24px',
+          fontWeight: 600,
+          lineHeight: 'normal',
+          marginBottom: '40px',
+          textAlign: 'center',
+          color: 'grey.1000',
+        }}
+      >
+        More Resources
+      </Typography>
```

Comment:
This wrapper & title probably should not be in the component. I think in the portal, this should be handled using the `SectionLayout` component just like other home pages. As a bonus, the `SectionLayout` will automatically handle hash routing as needed.